arenadata/readme.md

```diff
@@ -19,7 +23,14 @@ CI pushes docker images to the internal registry for each branch. We can pull it
 ## Full regression tests suite run
 
 We need to execute [../concourse/scripts/ic_gpdb.bash](../concourse/scripts/ic_gpdb.bash) in container to create demo cluster and run different test suites against it:
-
+for Ubuntu:
+```bash
+ docker run --name gpdb7_opt_on --rm -it -e TEST_OS=ubuntu \
+  -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=on' installcheck-world" \
+  --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_u22:latest \
+  /home/gpadmin/gpdb_src/concourse/scripts/ic_gpdb.bash
```

Comment:
I ran this command (on image without `segwalrep/max_slot_wal_keep_size` test) and got this error:
```sh
...
======================================================================
DIFF FILE: ../gpdb_src/src/bin/gpfdist/regress/regression.diffs
----------------------------------------------------------------------

diff -I HINT: -I CONTEXT: -I GP_IGNORE: -U3 /home/gpadmin/gpdb_src/src/bin/gpfdist/regress/expected/gpfdist_ssl.out /home/gpadmin/gpdb_src/src/bin/gpfdist/regress/results/gpfdist_ssl.out
--- /home/gpadmin/gpdb_src/src/bin/gpfdist/regress/expected/gpfdist_ssl.out	2024-10-22 09:23:08.832621335 +0000
+++ /home/gpadmin/gpdb_src/src/bin/gpfdist/regress/results/gpfdist_ssl.out	2024-10-22 09:23:08.834621342 +0000
@@ -70,6 +88,7 @@
 LOCATION ('gpfdists://127.0.0.1:7070/gpfdist_ssl/tbl2.tbl')
 FORMAT 'TEXT' (DELIMITER '|' );
 INSERT INTO tbl SELECT * FROM tbl_on_heap;
+ERROR:  error when connecting to gpfdist https://127.0.0.1:7070/gpfdist_ssl/tbl2.tbl, quit after 11 tries
 SELECT * FROM tbl_on_heap ORDER BY s1;
  s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
 -----+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
@@ -110,13 +129,11 @@
 LOCATION ('gpfdists://127.0.0.1:7070/gpfdist_ssl/tbl2.tbl')
 FORMAT 'TEXT' (DELIMITER '|' );
 INSERT INTO tbl_on_heap2 SELECT * FROM tbl2;
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 SELECT * FROM tbl_on_heap2 ORDER BY s1;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(3 rows)
+ s1 | s2 | s3 | dt | n1 | n2 | n3 | n4 | n5 | n6 | n7 
+----+----+----+----+----+----+----+----+----+----+----
+(0 rows)
 
 -- gpfdist_ssl case 3
 -- on same ETL server
@@ -130,23 +147,11 @@
 LOCATION ('gpfdists://127.0.0.1:7070/gpfdist_ssl/tbl1.tbl','gpfdists://127.0.0.1:7070/gpfdist_ssl/tbl2.tbl')
 FORMAT 'TEXT' (DELIMITER '|' );
 INSERT INTO tbl_on_heap SELECT * FROM tbl;
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 SELECT * FROM tbl_on_heap ORDER BY s1;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ddd | twod | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- eee | twoe | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- fff | twof | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ggg | twog | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- hhh | twoh | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- iii | twoi | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- jjj | twoj | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(13 rows)
+ s1 | s2 | s3 | dt | n1 | n2 | n3 | n4 | n5 | n6 | n7 
+----+----+----+----+----+----+----+----+----+----+----
+(0 rows)
 
 -- gpfdist_ssl case 4
 DROP TABLE IF EXISTS tbl_on_heap;
@@ -259,23 +285,18 @@
 
 GP_IGNORE:-- start_ignore
 GP_IGNORE:\! bash change_cert.bash recover_gpdb_root
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
 GP_IGNORE:SET verify_gpfdists_cert=on;
 GP_IGNORE:-- end_ignore
 SELECT * FROM tbl;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ddd | twod | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- eee | twoe | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- fff | twof | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ggg | twog | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- hhh | twoh | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- iii | twoi | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- jjj | twoj | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(10 rows)
-
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 GP_IGNORE:-- start_ignore
 GP_IGNORE:SET verify_gpfdists_cert=off;
 GP_IGNORE:WARNING:  verify_gpfdists_cert=off. Greenplum Database will stop validating the gpfdists SSL certificate for connections between segments and gpfdists
@@ -297,23 +318,18 @@
 
 GP_IGNORE:-- start_ignore
 GP_IGNORE:\! bash change_cert.bash recover_gpdb_client
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
 GP_IGNORE:SET verify_gpfdists_cert=on;
 GP_IGNORE:-- end_ignore
 SELECT * FROM tbl;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ddd | twod | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- eee | twoe | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- fff | twof | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ggg | twog | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- hhh | twoh | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- iii | twoi | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- jjj | twoj | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(10 rows)
-
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 GP_IGNORE:-- start_ignore
 GP_IGNORE:SET verify_gpfdists_cert=off;
 GP_IGNORE:WARNING:  verify_gpfdists_cert=off. Greenplum Database will stop validating the gpfdists SSL certificate for connections between segments and gpfdists
@@ -373,23 +410,18 @@
 
 GP_IGNORE:-- start_ignore
 GP_IGNORE:\! bash change_cert.bash recover_gpdb_root
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
+GP_IGNORE:client.crt.bak client.key.bak root.crt server.crt server.key
 GP_IGNORE:SET verify_gpfdists_cert=on;
 GP_IGNORE:-- end_ignore
 SELECT * FROM tbl;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ddd | twod | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- eee | twoe | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- fff | twof | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ggg | twog | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- hhh | twoh | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- iii | twoi | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- jjj | twoj | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(10 rows)
-
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 GP_IGNORE:-- start_ignore
 GP_IGNORE:SET verify_gpfdists_cert=off;
 GP_IGNORE:WARNING:  verify_gpfdists_cert=off. Greenplum Database will stop validating the gpfdists SSL certificate for connections between segments and gpfdists
@@ -411,23 +443,18 @@
 
 GP_IGNORE:-- start_ignore
 GP_IGNORE:\! bash change_cert.bash recover_gpdb_client
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
+GP_IGNORE:client.crt client.key root.crt server.crt server.key
 GP_IGNORE:SET verify_gpfdists_cert=on;
 GP_IGNORE:-- end_ignore
 SELECT * FROM tbl;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ddd | twod | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- eee | twoe | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- fff | twof | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ggg | twog | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- hhh | twoh | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- iii | twoi | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- jjj | twoj | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(10 rows)
-
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 GP_IGNORE:-- start_ignore
 GP_IGNORE:SET verify_gpfdists_cert=off;
 GP_IGNORE:WARNING:  verify_gpfdists_cert=off. Greenplum Database will stop validating the gpfdists SSL certificate for connections between segments and gpfdists
@@ -458,28 +485,15 @@
 GP_IGNORE:(1 row)
 GP_IGNORE:
 GP_IGNORE:select * from gpfdist_ssl_start_ca_verify;
-GP_IGNORE:       x       
-GP_IGNORE:---------------
-GP_IGNORE:  1777 gpfdist
+GP_IGNORE:        x        
+GP_IGNORE:-----------------
+GP_IGNORE:  237779 gpfdist
 GP_IGNORE:(1 row)
 GP_IGNORE:
 GP_IGNORE:SET verify_gpfdists_cert=on; 
 GP_IGNORE:-- end_ignore
 SELECT * FROM tbl;
- s1  |  s2  |   s3   |            dt            | n1 | n2  |   n3   |  n4   |   n5    |   n6    |     n7     
------+------+--------+--------------------------+----+-----+--------+-------+---------+---------+------------
- aaa | twoa | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- bbb | twob | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ccc | twoc | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ddd | twod | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- eee | twoe | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- fff | twof | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- ggg | twog | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- hhh | twoh | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- iii | twoi | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
- jjj | twoj | shpits | Wed Jun 01 12:30:30 2011 | 23 | 732 | 834567 | 45.67 | 789.123 | 7.12345 | 123.456789
-(10 rows)
-
+ERROR:  connection failed dummy_protocol://DUMMY_LOCATION
 GP_IGNORE:-- start_ignore
 GP_IGNORE:SET verify_gpfdists_cert=off;
 GP_IGNORE:WARNING:  verify_gpfdists_cert=off. Greenplum Database will stop validating the gpfdists SSL certificate for connections between segments and gpfdists


======================================================================
DIFF FILE: ../gpdb_src/src/test/isolation2/regression.diffs
----------------------------------------------------------------------

diff -I HINT: -I CONTEXT: -I GP_IGNORE: -U3 /home/gpadmin/gpdb_src/src/test/isolation2/expected/workfile_mgr_test.out /home/gpadmin/gpdb_src/src/test/isolation2/results/workfile_mgr_test.out
--- /home/gpadmin/gpdb_src/src/test/isolation2/expected/workfile_mgr_test.out	2024-10-22 08:45:05.577110393 +0000
+++ /home/gpadmin/gpdb_src/src/test/isolation2/results/workfile_mgr_test.out	2024-10-22 08:45:05.584110343 +0000
@@ -70,7 +69,7 @@
 1: DROP TABLESPACE IF EXISTS work_file_test_ts;
 DROP TABLESPACE
 1: CREATE TABLESPACE work_file_test_ts LOCATION '/home/gpadmin/gpdb_src/src/test/isolation2/testtablespace/workfile_mgr';
-CREATE TABLESPACE
+ERROR:  directory "/home/gpadmin/gpdb_src/src/test/isolation2/testtablespace/workfile_mgr/1/GPDB_7_302307241" already in use as a tablespace
 
 1: select gp_workfile_mgr_test('atomic_test', 0);
  gp_workfile_mgr_test 
@@ -147,13 +146,8 @@
  t                    
 (4 rows)
 1: select gp_workfile_mgr_test('workfile_made_in_temp_tablespace', 2000);
- gp_workfile_mgr_test 
-------------
- t                    
- t                    
- t                    
- t                    
-(4 rows)
+ERROR:  tablespace "work_file_test_ts" does not exist
+CONTEXT:  SQL function "gp_workfile_mgr_test" statement 1
 1: select gp_workfile_mgr_test('workfile_create_and_individual_cleanup_with_pinned_workfile_set', 2000);
  gp_workfile_mgr_test 
 ------------
@@ -164,7 +158,7 @@
 (4 rows)
 
 1: DROP TABLESPACE work_file_test_ts;
-DROP TABLESPACE
+ERROR:  tablespace "work_file_test_ts" does not exist
 
 GP_IGNORE:-- start_ignore
 GP_IGNORE:!\retcode gpconfig -r gp_workfile_max_entries --skipvalidation;
diff -I HINT: -I CONTEXT: -I GP_IGNORE: -U3 /home/gpadmin/gpdb_src/src/test/isolation2/expected/fts_doublefault.out /home/gpadmin/gpdb_src/src/test/isolation2/results/fts_doublefault.out
--- /home/gpadmin/gpdb_src/src/test/isolation2/expected/fts_doublefault.out	2024-10-22 09:00:48.775176720 +0000
+++ /home/gpadmin/gpdb_src/src/test/isolation2/results/fts_doublefault.out	2024-10-22 09:00:48.784176683 +0000
@@ -296,8 +924,9 @@
 -- kill the ftsprobe process.
 !\retcode pkill -f ftsprobe;
 GP_IGNORE:-- start_ignore
+GP_IGNORE:
 GP_IGNORE:-- end_ignore
-(exited with code 0)
+(exited with code -15)
 
 -- restarts ftsprobe, we should see another entry for content 0 doublefault into gp_configuration_history
 select gp_request_fts_probe_scan();


real	93m19.471s
user	71m13.424s
sys	31m5.954s
```

Reply:
is it possible that test is flaky?

Reply:
when I run
```sh
docker run --rm -it -e TEST_OS=ubuntu -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=off' installcheck-world" --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_u22:latest /home/gpadmin/gpdb_src/concourse/scripts/ic_gpdb.bash .
```
I got same errors

Reply:
This is a separate issue, not related to Ubuntu image itself.